s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -118,6 +116,41 @@ export class MapComponent implements OnInit, OnDestroy {
     });
   };
 
+  public getMapData(): Observable<MapData> {
+    this.working.emit(true);
+    const canvas: HTMLCanvasElement = this.map.getViewport().firstChild as HTMLCanvasElement;
+
+    const r = new ReplaySubject<MapData>(1);
+
+    // this will work only on new browsers!
```

Comment:
I see the third time a needed comment in my whole life. Keep it up!!! :) 

Could you add prefix like `IMPORTANT!!!` to focus other developers attention?